pkg/apis/networkservicemesh.io/v1/types.go

```diff
@@ -24,17 +26,34 @@ import (
 const (
 	NSMGroup           = "networkservicemesh.io"
 	NSMGroupVersion    = "v1"
-	NSMEPPlural        = "networkserviceendpoints"
-	FullNSMEPName      = NSMEPPlural + "." + NSMGroup
-	NSMChannelPlural   = "networkservicechannels"
-	FullNSMChannelName = NSMChannelPlural + "." + NSMGroup
-	NSMPlural          = "networkservices"
-	FullNSMName        = NSMPlural + "." + NSMGroup
+	NSMSpecPath        = "github.com/ligato/networkservicemesh/pkg/apis/" + NSMGroup
```

Comment:
Would it make sense to pull these out into a separate file in the same package so that regeneration doesn't require as much manual editing?

Reply:
This file itself is not auto-generated, I don't follow the comment. Can you please elaborate? Thanks!

Reply:
Ah... then I am mistaken :)